src/main/java/com/google/step/finscholar/data/Scholarship.java

```diff
@@ -0,0 +1,308 @@
+   */
+  public List<String> getEthnicityRaceRequirements() {
+    return ethnicityRaceRequirements;
+  }
+
+  /**
+   * The getter of the list of financial background requirements.
+   * @return The list of financial requirements.
+   */
+  public List<String> getFinancialRequirements() {
+    return financialRequirements;
+  }
+
+  /**
+   * The getter for gender restrictions of the scholarship.
+   * @return The list of gender requirements.
+   */
+  public List<String> getGenderRequirements() {
+    return genderRequirements;
+  }
+
+  /**
+   * The getter for geographic restrictions of the scholarship;
+   * @return The list of location requirements.
+   */
+  public List<String> getLocationRequirements() {
+    return locationRequirements;
+  }
+
+  /**
+   * The getter for nationality restrictions of the scholarship;
+   * @return The list of national origin requirements.
+   */
+  public List<String> getNationalOriginRequirements() {
+    return nationalOriginRequirements;
+  }
+
+  /**
+   * The getter of the number of years the scholarship is offered to one student.
+   * @return The duraton of the scholarship in year.
+   */
+  public Optional<Integer> getNumberOfYears() {
+    return numberOfYears;
+  }
+
+  /**
+   * The getter of the list of other requirements.
+   * @return The list of other requirements.
+   */
+  public List<String> getOtherRequirements() {
+    return otherRequirements;
+  }
+
+  /**
+   * The getter for the scholarship's name.
+   * @return The name of the scholarship.
+   */
+  public String getScholarshipName() {
+    return scholarshipName;
+  }
+
+  /**
+   * The getter for the UUID.
+   * @return the UUID of the scholarship.
+   */
+  public UUID getScholarshipUUID() {
+    return scholarshipUUID;
+  }
+  
+  /**
+   * The getter for the list of schools providing the scholarship.
+   * @return The schools list.
+   */
+  public List<UUID> getSchoolsList() {
+    return schoolsList;
+  }
+
+  /**
+   * The getter of the URL to the official website for the scholarship.
+   * @return The URL.
+   */
+  public String getURL() {
+    return URL;
+  }
+
+  /**
+   * The getter of the introduction.
+   * @return The optional containing introduction.
+   */
+  public Optional<String> getIntroduction() {
+    return introduction;
+  }
+
+  /**
+   * The getter of isRenewable.
+   * @return The Optional of boolean for isRenewable.
+   */
+  public Optional<Boolean> getIsRenewable() {
+    return isRenewable;
+  }
+
+  /**
+   * The setter for academicRequirements.
+   * @param academicRequirements
+   * @return The updated scholarship object.
+   */
+  public Scholarship setAcademicRequirements(List<String> academicRequirements) {
+    this.academicRequirements = academicRequirements;
+    return this;
+  }
+
+  /**
+   * The setter for amountPerYear.
+   * @param amount2
+   * @return The updated scholarship object.
+   */
+  public Scholarship setAmountPerYear(String amount2) {
```

Comment:
Coordinate with Michael here. Make sure if you use this patter, you use it for all your classes. In general, I think that would be a good idea.

Reply:
Yeah I just changed my setters to use "this"!

Reply:
Nice!